flashinfer/dllm/batch_block_extend.py

```
@@ -0,0 +1,664 @@
+    k_current: torch.Tensor,
+    v_current: torch.Tensor,
+    qo_indptr: torch.Tensor,
+    kv_curr_indptr: torch.Tensor,
+    paged_kv_cache: Optional[Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]]] = None,
+    paged_kv_indptr: Optional[torch.Tensor] = None,
+    paged_kv_indices: Optional[torch.Tensor] = None,
+    paged_kv_last_page_len: Optional[torch.Tensor] = None,
+    page_size: int = 16,
+    workspace_buffer: Optional[torch.Tensor] = None,
+    sm_scale: Optional[float] = None,
+    logits_soft_cap: float = 0.0,
```

Ruff (0.15.4)

[warning] Unused function argument: `logits_soft_cap`

(ARG001)